### PR TITLE
added albulm info

### PR DESCRIPTION
**Pull request by Brian-Kwong**
Updated albulm logic to allow it to fetch the current song information
IMG_5841

### CODE REVIEW

File: Application/Pawsitive-Studying/app/(Login)/security.js [hunk 1]
```diff
@@ -63,11 +63,7 @@ function login(username, password) {
 
 function savePassword(password) {
     return new Promise((resolve, reject) => {
-        if (
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Login)/security.js:66]: Removed Hardware Async and Auth Async

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx [hunk 2]
```diff
@@ -82,6 +83,13 @@ async function setupTrack(playlist, playlistName, songURLs) {
 
     await TrackPlayer.add(tracks);
 
+    if (tracks.length <= 0) return;
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:86]: Added to display what music is currently playing

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx [hunk 3]
```diff
@@ -127,6 +142,18 @@ const MusicPlayer = () => {
         }
     }
 
+    useTrackPlayerEvents(events, (event) => {
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:145]: Using the Music Player to set the song and display song info

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx [hunk 4]
```diff
@@ -168,13 +197,25 @@ const MusicPlayer = () => {
...
+                        currentSong.artwork === ""
+                            ? "https://cdn.pixabay.com/audio/2024/01/15/16-58-27-753_200x200.jpg"
+                            : `${currentSong.artwork}`,
                 }}
                 style={styles.albumCover}
             />
             <View style={styles.infoAndControls}>
-                <Text style={styles.songTitle}>Coverless book ( Lofi )</Text>
-                <Text style={styles.albumInfo}>AmbientAUDIOVISION</Text>
+                <Text style={styles.songTitle}>
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:208]: Display song info on music page

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx [hunk 5]
```diff
@@ -168,13 +197,25 @@ const MusicPlayer = () => {
...
-                    uri: "https://cdn.pixabay.com/audio/2024/01/15/16-58-27-753_200x200.jpg",
+                    uri:
+                        currentSong.artwork === ""
+                            ? "https://cdn.pixabay.com/audio/2024/01/15/16-58-27-753_200x200.jpg"
+                            : `${currentSong.artwork}`,
                 }}
                 style={styles.albumCover}
             />
             <View style={styles.infoAndControls}>
-                <Text style={styles.songTitle}>Coverless book ( Lofi )</Text>
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:176]: Was this just a template in place for a song?

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx [hunk 6]
```diff
@@ -67,12 +67,13 @@ async function newPlaylist(playlistID, playlists) {
     await TrackPlayer.add(tracks);
 }
 
-async function setupTrack(playlist, playlistName, songURLs) {
+async function setupTrack(playlist, playlistName, songURLs, setupCurrent) {
```
Comment (BrentNakashima) [inline Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:70]: added to have a current song

File: Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx (hunk 4 above)
Comment (Auroraz25) [reply Application/Pawsitive-Studying/app/(Main-App)/Player/player.jsx:211]: Displaying song information demonstrates this feature better

**Pull request merged**